app/renderer/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react";
+import "./Slider.scss";
+
+type SliderProps = {
+  max?: string;
+  value?: string;
+  children?: React.ReactNode;
+};
+
+type SliderStates = {
+  value: string;
+  max: string;
+};
+
+export class Slider extends React.Component<SliderProps, SliderStates> {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      value: props.value,
+      max: props.max,
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event: any): void {
+    this.setState({ value: event.target.value });
+  }
```

Comment:
If you needed `constructor` for binding you can declare you function as arrow function, so it will be 'binded' itself.
```javascript 
handleChange = (event: any): void => {
    this.setState({ value: event.target.value });
  }
```

https://reactjs.org/docs/faq-functions.html